includes/Demo_Content.php

```diff
@@ -0,0 +1,213 @@
+<?php
+/**
+ * Demo_Content class.
+ *
+ * Used for getting demo content.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Demo_Content class.
+ */
+class Demo_Content {
+	/**
+	 * Returns the title for the demo story.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Tips to make the most of Web Stories', 'web-stories' );
+	}
+
+	/**
+	 * Returns the content for the demo story.
+	 *
+	 * @return string
+	 */
+	public function get_content() {
+		$content = $this->load_demo_content_from_file();
+		$content = $this->localize_texts( $content );
+		$content = $this->update_assets_urls( $content );
+
```

Comment:
Why do these replacement before we decode the json?

Reply:
It's way easier to do this way.

Otherwise we have to:

1. Decode JSON content
1. Loop through **all**  fields to see if they have a placeholder
1. Make replacement

This is because placeholders can occur pretty much anywhere:

* in a text element's `content` property
* in an element's `link.url` and `link.desc` properties
* an element resource's `alt` field

Looping through all fields to check for that would be very slow.

Doing it on the raw JSON string before decoding is much faster, and due to the escaping everywhere shouldn't break the JSON.

Should the JSON string still get broken somehow, the `get_content()` method just returns an empty string